### ISSUE TITLE
Equation 19.11

### ISSUE THREAD

**Issue by AndreaPi** [udlbook/udlbook#66]
Eq. 19.11 seems wrong:

<img width="753" alt="image" src="https://user-images.githubusercontent.com/8233615/233724989-347fe7ba-7bcc-448f-8669-7a4f722bc072.png">

it should be like this, with $Pr(s_{t+1}|s_t, a_t) which doesn't multiply the reward:

<img width="712" alt="image" src="https://user-images.githubusercontent.com/8233615/233725124-6fb1b063-fe90-42a5-b8be-fe26842c3ff9.png">

BTW, it would be easier to write issues if we had access to the LaTeX formulas in the book.

**Comment by udlbook** [comment]
Thanks so much, Andrea.

I think it's not technically wrong, because the first term in the final brackets doesn't depend on s_{t+1}, so when we sum over these probabilities, the term ultimately just gets multiplied by one.   But I agree that this is confusing, so I've changed it.

Sometimes rewards are defined as r[s_t,a_t,s_{t+1}].  If this was the case, we'd have to leave it where it is.

Let me know if you find any more.  I really appreciate your continued help.

**Comment by AndreaPi** [comment]
> I think it's not technically wrong, because the first term in the final brackets doesn't depend on s_{t+1}, so when we sum over these probabilities, the term ultimately just gets multiplied by one. 

Indeed, I never said it was wrong. Actually, in my comment I noted that equations 19.11 and 19.9 are equivalent:

> The two equations are equivalent, since 
>
> $$\sum_{s_{t_+1}}Pr(s_{t+1}|s_t, a_t)=1$$

It's just that using two different notations (albeit **equivalent**) may be confusing for readers less acquainted with Bellmann's equation. I suggest two possible alternatives:

- keep equation 19.11 as it is, but explain that they are equivalent  by adding your explanatory text:

    > the first term in the final brackets doesn't depend on s_{t+1}, so when we sum over these probabilities, the term ultimately just gets multiplied by one

- alternatively, add an exercise asking to prove the equivalence between 19.11 and 19.9. For example, after introducing eq. 19.11 you may write

   > While equation 19.11 and 19.9 look different, they are actually equivalent. We ask the reader to prove this equivalence in exercise XXX

**Comment by udlbook** [comment]
I actually already changed it to what you suggested in the first message!
